drivers/SmartThings/zigbee-sensor/src/init.lua

```diff
@@ -132,6 +99,14 @@ local zigbee_generic_sensor_template = {
     doConfigure = do_configure,
     infoChanged = info_changed
   },
+  sub_drivers = {
+    require("contact"),
+    require("motion"),
+    require("waterleak"),
+    require("motion-illuminance"),
+    require("meian"),
+    require("ezviz")
```

Comment:
How about changing to 'meian-button' / 'ezviz-button' for the clarity? (including the folder name of each sub drivers)